### PR TITLE
Create SM.MS.xml

### PR DESCRIPTION
**Pull request by koops76**

### CODE REVIEW

**Comment by gloomy-ghost** [comment]
https://hstspreload.com/api/v1/status/sm.ms preloaded

**Comment by koops76** [comment]
@gloomy-ghost Fixed.

**Comment by koops76** [comment]
@gloomy-ghost https://hstspreload.com/api/v1/status/sm.ms returns `Forbidden` for me.

File: src/chrome/content/rules/ooo.0o0.ooo.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<ruleset name="ooo.0o0.ooo">
+	<target host="ooo.0o0.ooo" />
+
+	<securecookie host=".+" name=".+" />
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/ooo.0o0.ooo.xml]: Does this domain set any cookies?

Comment (koops76) [reply src/chrome/content/rules/ooo.0o0.ooo.xml]: I'm not sure, but if it doesn't, `securecookie` will simply do nothing.

**Comment by gloomy-ghost** [comment]
Sometimes I get 403 as well... At least we can look up the browser source code when hstspreload.com is not available.

**Comment by Hainish** [comment]
I can't access this:

```shell
(.venv)user@mutt:~$ curl https://ooo.0o0.ooo
curl: (7) Failed to connect to ooo.0o0.ooo port 443: No route to host
```

**Comment by lilydjwg** [comment]
@Hainish `No route to host` usually means that you can't access that IP. Can you access its HTTP version at the same time?

**Pull request merged**